src/components/dropdown/group/bl-dropdown-group.css

```
@@ -0,0 +1,38 @@
+:host {
+    position: relative;
+    width: 100%;
+}
+
+.dropdown-group {
+    display: flex;
+    flex-direction: column;
+    gap: var(--bl-size-xs);
+}
+
+.caption {
+    font-size: var(--bl-font-size-xs);
+    font-weight: var(--bl-font-weight-medium);
+    line-height: var(--bl-font-size-s);
+    color: var(--bl-color-content-secondary);
+}
+
+/* adding seperator before and after of element, except not first child and last child of the dropdown */
+:host(:not(:first-child))::before,
+:host(:not(:last-child))::after {
+    position: absolute;
+    content: '';
+    width: 100%;
+    border-bottom: 1px solid var(--bl-color-border);
+}
```

`border-bottom` and `border-top` would be enough here, I think. Like

```suggestion
:host(:not(:first-child))::before {
    border-bottom: 1px solid var(--bl-color-border);
}

:host(:not(:last-child))::after {
    border-top: 1px solid var(--bl-color-border);
}
```

---

🤔 It wouldn't work without those.
```css
position: absolute;
content: '';
width: 100%;
```
However `::after` and `::before` is not necessary here. I will delete those too to simplify like you suggested.

---

I just felt our aim is as same as adding a border-top or border-bottom to the group element here, isn't it?

Without `::after` and `::before`, `content` will not do anything.

---

Yeah, we are completely in the same page here.